CHANGELOG-1.0.md

```diff
@@ -0,0 +1,138 @@
+# Release notes for 
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v0.5.0
+
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Do not call controller-expand if volume is in-use and can not be expanded while in-use. New RBAC rules for listing pods are required. ([#86](https://github.com/kubernetes-csi/external-resizer/pull/86), [@gnufied](https://github.com/gnufied))
+- Adds a new flag `handle-volume-inuse-error` which can be set to `false` if a CSI driver does not want in-use check to be performed before issuing CSI expansion RPC calls([#89](https://github.com/kubernetes-csi/external-resizer/pull/89), [@saikat-royc](https://github.com/saikat-royc))
+- The `--csiTimeout` flag of external-resizer is obsoleted with this release. Instead `--timeout` flag has been introduced for the same purpose. The   timeout value  accommodates majority of `ControllerExpandVolume` calls. `timeout` value defaults to 10 seconds. From now on, the deployment  templates or  artifacts has to make use of `--timeout` flag instead of obsoleted `--csiTimeout`  flag. ([#82](https://github.com/kubernetes-csi/external-resizer/pull/82), [@humblec](https://github.com/humblec))
```

Comment:
What does "the timeout value accomodates majority of ControllerExpandVolume calls" mean?

Reply:
fixed it.